static/third/marked/lib/marked.js

```diff
@@ -714,6 +715,13 @@ InlineLexer.prototype.output = function(src) {
       continue;
     }
 
+    // italic
+    if (cap = this.rules.italic.exec(src)) {
+      src = src.substring(cap[0].length);
+      out += this.renderer.italic(this.output(cap[1]));
+      continue;
+    }
+
```

Comment:
Just a check: Would using the `em` logic in the next block work instead?  I think technically in markdown it's supposed to do `<em>`, so maybe we don't need to add this code but just should be adjusting the above.